### PR TITLE
LPS-149618 Add condition when ddmFormFieldType is fieldset

### PR DESCRIPTION
**Pull request by danvb112**
Related Issue: https://issues.liferay.com/browse/LPS-149618

### CODE REVIEW

**Comment by liferay-forms** [comment]
ci:test:sf

**Comment by liferay-forms** [comment]
Hi, you don’t need to run the CI commands anymore, I’ll do it for you and let you know if any issue happens.

**Comment by liferay-forms** [comment]
ci:test:relevant

**Comment by liferay-forms** [comment]
ci:test:relevant

File: modules/apps/object/object-dynamic-data-mapping/src/main/java/com/liferay/object/dynamic/data/mapping/internal/storage/ObjectDDMStorageAdapter.java [hunk 1]
```diff
@@ -315,7 +315,9 @@ private long _getObjectDefinitionId(
 			DDMFormField ddmFormField = ddmFormFieldsMap.get(
 				ddmFormFieldValue.getName());
 
-			if (ddmFormField.isTransient()) {
+			if (ddmFormField.isTransient() &&
+				!StringUtil.equals(ddmFormField.getType(), "fieldset")) {
```
Comment (gabrielwas) [inline modules/apps/object/object-dynamic-data-mapping/src/main/java/com/liferay/object/dynamic/data/mapping/internal/storage/ObjectDDMStorageAdapter.java:319]: @danvb112 can you use DDMFormFieldTypeConstants.FIELDSET here?

**Review by gabrielwas — CHANGES_REQUESTED** [review]

**Comment by danvb112** [comment]
ci:test:sf

**Comment by liferay-forms** [comment]
ci:test:relevant

**Comment by gabrielwas** [comment]
LGTM :D

**Review by gabrielwas — APPROVED** [review]

**Comment by liferay-forms** [comment]
Hey @danvb112, can you please check if these failures are related to your changes? Thanks

**Comment by liferay-forms** [comment]
Hey @danvb112, can you please check if these failures are related to your changes? Thanks

**Comment by danvb112** [comment]
ci:test:relevant

**Pull request closed without merging**